firmware/framebuffer.h

```diff
@@ -40,15 +41,19 @@
  *
  */
 #define FRAMEBUFFER_OFFSET		0x01000000
```

Comment:
This should probably be based on the value of `MAIN_RAM_OFFSET` or similar?

Could actually use the linker script to place these buffers...

Reply:
I think that those offsets are already internally based on `MAIN_RAM_BASE` position because when CPU accesses those buffers it [adds](https://github.com/timvideos/litex-buildenv/blob/master/firmware/pattern.c#L142) the `MAIN_RAM_BASE` to the offset.
And DMA controllers [take](https://github.com/timvideos/litex-buildenv/blob/master/firmware/processor.c#L726) only the offset value which means that they already use addresses relative to main memory